src/main/java/se/kth/hopsworks/controller/ProjectController.java

```diff
@@ -757,4 +764,39 @@ private String extractProjectName(String path) {
 
     return path.substring(startIndex + 1, endIndex);
   }
+
+  public void setQuota(Path src, long diskspaceQuota)
+      throws IOException {
+    dfsSingleton.getDfsOps().setQuota(src, diskspaceQuota);
+  }
+
+  public void setQuota(String projectname, long diskspaceQuota)
+      throws IOException {
+    String rootDir = settings.DIR_ROOT;
+    String fullProjectPath = File.separator + rootDir + File.separator
+        + projectname;
+    String project = this.extractProjectName(fullProjectPath + File.separator);
+    String projectPath = File.separator + rootDir + File.separator + project;
+    dfsSingleton.getDfsOps().setQuota(new Path(projectPath), diskspaceQuota);
+  }
+
+  //Get quota in GB
+  public long getQuota(String projectname) throws IOException {
+    String rootDir = settings.DIR_ROOT;
+    String fullProjectPath = File.separator + rootDir + File.separator
+        + projectname;
```

Comment:
It is correct, you mean Projects right? I will make a method for it though you are right